projects/aws/image-builder/builder/builder.go

```diff
@@ -115,43 +114,19 @@ func (b *BuildOptions) BuildImage() {
 			}
 		}
 
-		if b.Os == Ubuntu {
```

Comment:
I think we should actually check the existence of the `/usr/share/edk2/ovmf/OVMF_CODE.fd` and the `OVMF.fd` file and depending on which exists, make sure our image-builder code is passing that. this way al2 and ubuntu/rhel hosts should work for this process. thoughts?